F21/amirsalari/convert.py

```diff
@@ -0,0 +1,123 @@
+import os.path
+import sys
+
+
+def convert(test_word):
+    if(test_word[1] in ["en", "en-US", "en-IE", "en-Latn"]):
+        lowercase_word = test_word[0].lower()
+        return lowercase_word
+
+    # Specification 1
+    if(test_word[1] in ["tr", "az"]):
+        lowercase_word = test_word[0]
+        lowercase_word = lowercase_word.replace('\u0049', '\u0131')  # I ı
+        lowercase_word = lowercase_word.replace('\u011E', '\u011F')  # Ğ ğ
+        lowercase_word = lowercase_word.replace('\u00C7', '\u00E7')  # Ç ç
+        lowercase_word = lowercase_word.replace('\u00D6', '\u00F6')  # Ö ö
+        lowercase_word = lowercase_word.replace('\u00DC', '\u00FC')  # Ü ü
+        lowercase_word = lowercase_word.replace('\u015E', '\u015F')  # Ş ş
+        lowercase_word = lowercase_word.lower()
+        return lowercase_word
+    # Specification 2
+    if(test_word[1] in ["ga", "ga-IE"]):
+        lowercase_word = test_word[0]
+        first = test_word[0][0]
+        second = test_word[0][1]
+        first_letters = ['n', 't']
+        second_letters = ['A', 'E', 'I', 'O', 'U',
+                          '\u00C1', '\u00C9', '\u00CD', '\u00D3', '\u00D5', '\u00DA']
+        if (first in first_letters):
+            '''Unicode	Upper   Unicode	    	Lower	Character Name
+                00C1	Á	    00E1	        á	        A acute
+                00C9	É	    00E9	        é	        E acute
+                00CD	Í	    00ED	        í	        I acute
+                00D3	Ó	    00F3	        ó	        O acute
+                00D5	Õ	    00F5	        ó	        O acute
+                00DA	Ú	    00FA	        ú	        U acute'''
+
+            if(second in second_letters):
+                lowercase_word = lowercase_word.lower()
+                lowercase_word = lowercase_word[:1] + "-" + lowercase_word[1:]
```

Comment:
Some tests are failing (tests 9,18,19) because a '-' is not being correctly added into the word.